eng/testing/performance/microbenchmarks.proj

```diff
@@ -81,6 +80,7 @@
 
   <PropertyGroup>
     <PartitionCount>30</PartitionCount>
+    <EnableAzurePipelinesReporter>false</EnableAzurePipelinesReporter>
```

Comment:
Currently due to some helix recent change, helix reporter crashes due to perf test alters PYTHONPATH. Set EnableAzurePipelinesReporter to false since perf test doesn't use helix reporter to upload test results, instead using perf specific scripts.